DataConnectors/Okta Single Sign-On/azuredeploy_OktaSingleSignOn_API_FunctionApp_V2.json

```diff
@@ -4,6 +4,8 @@
       "parameters": {
           "FunctionName": {
               "defaultValue": "OktaSSO",
+              "minLength": 1,
+              "maxLength": 10,
```

Comment:
uniqueString size  -- 13
storage account size limitation -- 24

function name length can 11, can you test tit as well?